web/react/components/textbox.jsx

```diff
@@ -303,7 +303,19 @@ export default class Textbox extends React.Component {
     }
 
     render() {
-        const previewLinkVisible = this.props.messageText.length > 0;
+        let previewLink = null;
+        if (Utils.featureToggle('#1389')) {
```

Comment:
Yep, that's the github-issue id.
Constants is probably a better place to put those, agree